Content.Server/Pinpointer/NavMapSystem.cs

```diff
@@ -452,13 +476,11 @@ public string GetNearestBeaconString(MapCoordinates coordinates)
             ? Loc.GetString("nav-beacon-pos-format-direction-mod-far")
             : string.Empty;
 
-        // we can null suppress the text being null because TryGetNearestVisibleStationBeacon always gives us a beacon with not-null text.
+        // we can null suppress the text being null because TRyGetNearestVisibleStationBeacon always gives us a beacon with not-null text.
         return Loc.GetString("nav-beacon-pos-format-direction",
             ("modifier", modifier),
             ("direction", ContentLocalizationManager.FormatDirection(adjustedDir).ToLowerInvariant()),
             ("color", beacon.Value.Comp.Color),
             ("marker", beacon.Value.Comp.Text!));
     }
-
-    #endregion
```

Comment:
Don't remove #endregion, its whats throwing errors for you.

Reply:
Damn, I didn't even realize I did that. Thanks!